pkg/stream/pipeline.go

```diff
@@ -149,11 +151,17 @@ func (p *Pipeline) process(ctx context.Context, force bool) {
 	}()
 
 	p.ticker.Stop()
-	messages, err := p.callback.Process(ctx, p.batch)
 
-	if err != nil {
-		p.logger.Error(err, "could not process the batch")
-		return
+	var err error
+	var messages = p.batch
```

Comment:
why do you write it that way? Don't we usually do it like this: `messages := p.batch` ?